### PR TITLE
LPD-30647 test

### PR DESCRIPTION
**Pull request by joshuacords**
Follow up tests for: https://github.com/brianchandotcom/liferay-portal/pull/152641#issuecomment-2249572859

**Author:** @joshuacords

### CODE REVIEW

**Comment by joshuacords** [comment]
ci:test:sf

**Comment by joshuacords** [comment]
ci:test:search

**Comment by joshuacords** [comment]
ci:test:search-opensearch2

**Comment by joshuacords** [comment]
ci:report:45972690

**Comment by joshuacords** [comment]
ci:test:relevant

**Comment by joshuacords** [comment]
ci:report:45913884

File: modules/apps/portal-search-opensearch2/portal-search-opensearch2-impl/src/test/java/com/liferay/portal/search/opensearch2/internal/OpenSearchIndexWriterTest.java [hunk 1]
```diff
@@ -106,6 +106,21 @@ public void testPartiallyUpdateDocumentDoesNotRemoveFields()
 		_assertOnlyOne(Field.TITLE, "text");
 	}
 
+	@Test
+	public void testPartiallyUpdateUpsertsMissingDocument()
```
Comment (gustavolimav) [inline modules/apps/portal-search-opensearch2/portal-search-opensearch2-impl/src/test/java/com/liferay/portal/search/opensearch2/internal/OpenSearchIndexWriterTest.java:110]: Maybe rename `testPartiallyUpdateDocumentUpsertsMissingDocument` as we are testing the `partiallyUpdateDocument` method right?

**Review by gustavolimav — CHANGES_REQUESTED** [review]

**Comment by gustavolimav** [comment]
Other than that code LGTM!

**Comment by joshuacords** [comment]
ci:forward

**Pull request closed without merging**